### PR TITLE
Add annotation to SDK

### PR DESCRIPTION
**Pull request by omid**

### CODE REVIEW

File: src/Model/VideoManager.php [hunk 1]
```diff
@@ -2,25 +2,24 @@
...
+     */
     private $id;
-    private $name;
 
     /**
-     * @param $name
-     * @param $id
+     * @Type("string")
      */
-    public function __construct($name, $id)
```
Comment (EyesOnlyNet) [inline src/Model/VideoManager.php:19]: I do not know why this constructor was there but maybe analytics need it - have you checked it!?

File: src/Model/VideoManager.php [hunk 2]
```diff
@@ -2,25 +2,24 @@
...
+
 /**
  * @author Steve Reichenbach <steve.reichenbach@movingimage.com>
  *
  * @codeCoverageIgnore
  */
 class VideoManager
 {
+    /**
+     * @Type("string")
```
Comment (EyesOnlyNet) [inline src/Model/VideoManager.php:15]: The annotations are already defined here: https://github.com/MovingImage24/videomanager-pro-sdk/blob/master/resources/serializer/VideoManager.xml

File: src/Model/Login.php [hunk 3]
```diff
@@ -2,16 +2,33 @@
...
+
 /**
  * @author Steve Reichenbach <steve.reichenbach@movingimage.com>
  *
  * @codeCoverageIgnore
  */
 class Login
 {
+    /**
+     * @Type("string")
```
Comment (EyesOnlyNet) [inline src/Model/Login.php:15]: These annotations are already defined: https://github.com/MovingImage24/videomanager-pro-sdk/blob/master/resources/serializer/Login.xml

**Comment by EyesOnlyNet** [comment]
We want to provide the serialization-configuration without annotations. Makes the code better readable and decrease the dependency to another library.

File: src/Model/VideoManager.php (hunk 1 above)
Comment (omid) [reply src/Model/VideoManager.php:19]: Yes, I've checked. Only in some tests.
All phpunit tests are passed, but I didn't investigate why those tests are still working!

File: src/Model/Login.php (hunk 3 above)
Comment (omid) [reply src/Model/Login.php:15]: You are right. I cannot make it work with those XMLs.

**Pull request closed without merging**